.github/workflows/workflow.yaml

```diff
@@ -19,6 +22,18 @@ on:
       runs-on:
         type: string
         default: self-hosted
+      trivy-job-enabled:
+        type: boolean
+        default: true
+      trivy-severity:
+        type: string
+        default: MEDIUM,HIGH,CRITICAL
+      trivy-trivyignores:
```

Comment:
it would be better to name this `trivy-ignore-files` since it is actually a list of ignore files, and not a list of issues to ignore as the name can suggest now

Reply:
The thing with `ingore-files` is that it collides with the Terraform check argument that actually *ignores files*, while this is a list of *ignore-files*. `trivyignores` was a way out here, as this is the name of the input variable used in the actual aquasec/trivy action.

Reply:
but prefixing with `trivy` should not cause this conflict?

Reply:
I think `trivy-ignore-files` is quite sufficient, it says to me list of ignore files. `trivy-files-ignore` would be list of files to ignore.

Reply:
Okay, I can agree. Putting on a list of changes for `v2` to rename the terraform arg to be `terraform-files-ignore`